sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateWorksetDao.java

```diff
@@ -0,0 +1,157 @@
+package uk.ac.bbsrc.tgac.miso.persistence.impl;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.MatchMode;
+import org.hibernate.criterion.Restrictions;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Workset;
+import uk.ac.bbsrc.tgac.miso.core.store.WorksetStore;
+import uk.ac.bbsrc.tgac.miso.core.util.DateType;
+import uk.ac.bbsrc.tgac.miso.sqlstore.util.DbUtils;
+
+@Repository
+@Transactional(rollbackFor = Exception.class)
+public class HibernateWorksetDao implements WorksetStore, HibernatePaginatedDataSource<Workset> {
+
+  private static final String[] SEARCH_PROPERTIES = new String[] { "alias", "description" };
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "alias" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)